app/views/uniform/fragments/returnsCYA.scala.html

```diff
@@ -38,27 +39,33 @@
 
     <tbody>
         @for((lineKey, litres, multiplier) <- lineItems) {
-            <tr>
-                <td rowspan="2">@Messages(s"$key.$lineKey")</td>
-                <td>@Messages(s"$key.low")</td>
-                <td>@{f"${litres._1}%,d"}</td>
-                <td class="no-minus-wrapping">@formatMoney(costLower * litres._1 * multiplier)</td>
-                <td>
-                    <a href="@lineKey" class="returns-change-link">
-                        @Messages(s"$key.change") <span class="visuallyhidden">@Messages(s"$key.change.$lineKey")</span>
-                    </a>
-                </td>
-            </tr>
-            <tr>
-                <td>@Messages(s"$key.high")</td>
-                <td>@{f"${litres._2}%,d"}</td>
-                <td class="no-minus-wrapping"> @formatMoney(costHigher * litres._2 * multiplier)</td>
-                <td>
-                    <a href="@lineKey" class="returns-change-link">
-                        @Messages(s"$key.change") <span class="visuallyhidden">@Messages(s"$key.change.$lineKey")</span>
-                    </a>
-                </td>
-            </tr>
+            @if(lineKey != "own-brands-packaged-at-own-sites" || !isSmallProducer) {
```

Comment:
I think this might be better handled in the controller - so that line item is never passed into the view. Then you would have no need to add in an extra parameter for the view too.